docs/rest.md

```diff
@@ -303,12 +319,12 @@ In addition to the fields described in `/v2/find`:
 
 [//]: # keep the fields sorted!
 
-* `channel`: which channel the package is currently tracking.
+* `apps`: JSON array of apps the snap provides. Each app has a `name` field to name a binary this app provides.
 * `installed-size`: how much space the snap itself (not its data) uses.
 * `install-date`: the date and time when the snap was installed.
-* `status`: can be either `installed` or `active` (i.e. is current).
+* `trymode`: true if the app was installed in try mode.
```

Comment:
Please copy the devmode description here.

Also, why was status dropped? Note that the description differs.